Source/items.h

```diff
@@ -357,6 +357,16 @@ struct Item {
 		}
 	}
 
+	[[nodiscard]] bool IsScroll() const

```

Comment:
Do we really need this `[[nodiscard]]`? What does it do?

Reply:
Errors if the return value is not used by the caller. Recommended for getter methods by clang-tidy.

Reply:
Interesting. I guess it would make sense to omit it only when there are out arguments and the user might potentially ignore the `bool` return and stuff like that.

Thanks for clarifying.